components/navbar/navbar.jsx

```diff
@@ -732,10 +732,12 @@ export default class Navbar extends React.Component {
                             {deleteChannelOption}
                             {leaveChannelOption}
                             {toggleFavoriteOption}
-                            <MobileChannelHeaderPlug
```

Comment:
I think you need to maintain these two `MobileChannelHeaderPlug` since one is for dropdown and the other one is for button.

Reply:
We don't need that icon in the navbar. I have placed it in the dropdown menu. The same thing should not exist in both places.